cli/src/logger/mod.rs

```diff
@@ -1,4 +1,4 @@
-#[repr(u8)]
+#[repr(u64)]
```

Comment:
![you crazy son of a bitch, you did it](https://cdn3-www.musicfeeds.com.au/assets/uploads/he-did-it.gif)